### PR TITLE
add back tooltip-triangle icon

### PR DESCRIPTION
**Pull request by rasom**
status: ready

### CODE REVIEW

File: src/status_im/ui/components/tooltip/views.cljs [hunk 1]
```diff
@@ -17,17 +17,20 @@
      [react/animated-view {:style (styles/tooltip-animated bottom-anim-value opacity-value)}
       [react/view (styles/tooltip-text-container color)
        [react/text {:style (styles/tooltip-text font-size)} label]]
-      [vector-icons/icon :icons/tooltip-triangle {:color color :style styles/tooltip-triangle}]]]))
+      [vector-icons/icon :icons/tooltip-triangle (merge
```
Comment (cammellos) [inline src/status_im/ui/components/tooltip/views.cljs]: `assoc`?

File: src/status_im/ui/components/tooltip/views.cljs [hunk 2]
```diff
@@ -17,17 +17,20 @@
...
 (views/defview bottom-tooltip-info [label on-close]
   (views/letsubs [bottom-anim-value (animation/create-value -150)
                   opacity-value     (animation/create-value 0)]
     {:component-did-mount (animations/animate-tooltip -150 bottom-anim-value opacity-value -10)}
     [react/view styles/bottom-tooltip-container
      [react/animated-view {:style (styles/tooltip-animated bottom-anim-value opacity-value)}
-      [vector-icons/icon :icons/tooltip-triangle {:color           colors/gray
-                                                  :style           styles/tooltip-triangle
-                                                  :container-style {:transform [{:rotate "180deg"}]}}]
+      [vector-icons/icon :icons/tooltip-triangle (merge
```
Comment (cammellos) [inline src/status_im/ui/components/tooltip/views.cljs]: `assoc` here?

**Review by cammellos — APPROVED** [review]

File: src/status_im/ui/components/tooltip/views.cljs (hunk 1 above)
Comment (rasom) [reply src/status_im/ui/components/tooltip/views.cljs]: yeah, it can be `assoc` as well

**Review by jeluard — APPROVED** [review]

**Comment by asemiankevich** [comment]
@rasom where this icon supposed to be in the app?

**Comment by rasom** [comment]
@asemiankevich in tooltips. For instance when you enter a wrong password you will see one. In nightly version it looks differently because an icon is missing.

**Comment by asemiankevich** [comment]
How it is in nightly:

image


How it is branch:

image

**Comment by rasom** [comment]
@asemiankevich yep, so in PR version it looks the same way as in release

**Pull request merged**